src/library/Server/Manager/Directadmin.php

```diff
@@ -315,152 +394,176 @@ public function modifyAccount(Server_Account $a)
-            'timeout'       => 60,
+        $http_client = $this->getHttpClient()->withOptions([
             'auth_basic'    => [ $this->_config['username'], $this->_config['password'] ],
+            'timeout'       => 60,
+            'verify_host'   => false,
+            'verify_peer'   => false,
         ]);
 
-        $url = $protocol . $host . ':'. $this -> _getPort().'/' . $command . '?' . $fieldstring;
+        $url = $protocol . $host . ':'. $this -> _getPort().'/CMD_' . $command . '?' . $field_string;
         $this->getLog()->debug($url);
 
         try {
-            if($post) {
-                $request = $httpClient->request('POST', $url, [
-                    'body'  => $fields,
-                ]);
-            } else {
-                $request = $httpClient->request('GET', $url);
-            }
-
+            // If it's a POST request, include the fields in the request body
+            $request = $http_client->request($post ? 'POST' : 'GET', $url, $post ? ['body' => $fields] : []);
             $data = $request->getContent();
         } catch (\Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface |
                  \Symfony\Contracts\HttpClient\Exception\HttpExceptionInterface $error) {
-            $e = new Server_Exception('HttpClientException: :error', [':error' => $error->getMessage()]);
-            $this->getLog()->err($e);
-            throw $e;
+            $exception = new Server_Exception('HttpClientException: :error', [':error' => $error->getMessage()]);
+            $this->getLog()->err($exception);
+            throw $exception;
         }
 
-        if(strlen(strstr($data, 'DirectAdmin Login')) > 0) {
+        if(strlen(strstr($data, '<!doctype html>')) > 0 || strlen(strstr($data, 'DirectAdmin Login')) > 0) {
             throw new Server_Exception('Failed to connect to the :type: server. Please verify your credentials and configuration', [':type:' => 'DirectAdmin']);
         }
 
         if(strlen(strstr($data, "The request you've made cannot be executed because it does not exist in your authority level")) > 0) {
             throw new Server_Exception('Server Manager DirectAdmin Error: "The request you have made cannot be executed because it does not exist in your authority level"');
         }
 
-        $r = $this->_parseResponse($data);
+        $response = $this->_parseResponse($data);
 
-        if(isset($r['error']) && $r['error'] == 1) {
-            $placeholders = [':action:' => $command, ':type:' => 'DirectAdmin'];
-            throw new Server_Exception('Failed to :action: on the :type: server, check the error logs for further details', $placeholders);
+        if(isset($response['error']) && $response['error'] == 1) {
+            $placeholders = [':action:' => $command, ':type:' => 'DirectAdmin', ':error:' => $response['text'] . ': ' . $response['details']];
+            throw new Server_Exception('Failed to :action: on the :type: server: :error:', $placeholders);
```

Comment:
I'm typically not a big fan on sending errors to the front-end like this, clients will also get shown this message and sometimes returned errors can be a bit too.. *showy* if that makes sense.

Reply:
Completely agree, apologies for the oversight! I'll adjust this, and log the error internally